docs/Reply.md

```diff
@@ -177,15 +177,20 @@ fastify.get('/', function (request, reply) {
   reply.send(httpErrors.Gone())
 })
 ```
+By default Fastify will log the error depending on the `statusCode`:
+- for `400 <= statusCode < 500` the error will be logged with level `info`
+- for `statusCode >= 500` the error will be logged with level `error`
 
-If you want to completely customize the error response, checkout [`setErrorHandler`](https://github.com/fastify/fastify/blob/master/docs/Server.md#seterrorhandler) API.
+If you want to completely customize the error handling, checkout [`setErrorHandler`](https://github.com/fastify/fastify/blob/master/docs/Server.md#seterrorhandler) API.<br>
+*Note: you are responsibile for logging when setting the error handler*
 
 The not found errors generated by the router will use the  [`setNotFoundHandler`](https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md#setnotfoundhandler)
 
 API:
 
 ```js
-fastify.setNotFoundHandler(function (request, reply) {
+fastify.setNotFoundHandler(function (error, request, reply) {
+  request.log.error(error)
```

Comment:
this is the nof found handler, not the error handler.

Reply:
jeez! I will revert it